### PR TITLE
Upgrade to Bevy 0.15

### PR DESCRIPTION
**Pull request by bonsairobo**
- [x] Make the crate itself compile without warnings
- [x] Work around a bug in `bevy_gltf`
- [x] Make the examples run (with deprecation warnings)
- [x] Fix deprecation warnings
- [x] Bump crate version
- [x] Update the bevy version support table
- [x] Update documentation (replace `bevy_mod_picking` with `bevy::picking`)

### CODE REVIEW

File: src/extensions/anchor_indicator.rs [hunk 1]
```diff
@@ -80,8 +80,7 @@ pub fn draw_anchor(
             let arm_length = 0.4;
 
             gizmos.circle(
-                anchor_world,
-                Dir3::new_unchecked(cam_transform.forward().normalize()),
+                Isometry3d::new(anchor_world, cam_transform.rotation()),
```
Comment (bonsairobo) [inline src/extensions/anchor_indicator.rs:83]: I'm not sure this is right, but it seems to work as expected in the cad example.

**Comment by bonsairobo** [comment]
The migration is done at a surface level, but the picking seems broken in all examples. I might be missing something about switching from `bevy_mod_picking` to `bevy::picking`.

There are some other things that seem a little off, but they might just be fixed by fixing the picking issue.

**Comment by bonsairobo** [comment]
The `map` example has some very offputting temoral AA. I wonder if it's just misconfigured?

**Comment by aevyrie** [comment]
Thank you @bonsairobo!

**Pull request merged**